### PR TITLE
Fixed frint-router-react tests for React 16 compatibility

### PR DESCRIPTION
**Pull request by ascariandrea**
I've fixed tests by following the [migration guide for enzyme v2 to v3](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md).

### CODE REVIEW

**Comment by fahad19** [comment]
@ascariandrea: great effort! thanks for contributing!

there's only one unit test left that needs to be taken care of :) https://travis-ci.org/Travix-International/frint/builds/289411656#L2541

File: packages/frint-react/src/components/Region.spec.js [hunk 1]
```diff
@@ -133,6 +133,7 @@ describe('frint-react › components › Region', function () {
     );
 
     // verify
+    console.warn.restore(); // eslint-disable-line no-console
```
Comment (ascariandrea) [inline packages/frint-react/src/components/Region.spec.js]: Not sure why the method was already stubbed, do you @fahad19 have any clue on this?

Comment (ascariandrea) [reply packages/frint-react/src/components/Region.spec.js]: Nevermind, it was something related to my local env.

**Comment by fahad19** [comment]
@ascariandrea: many thanks for taking care of it! 🎉 

 I have merged the PR in #340, and will ultimately land in `master` very soon.

**Comment by ascariandrea** [comment]
You're welcome :)

I'm going to take care of the examples now 😎

**Pull request merged**